lib/mortar/list_command.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require_relative "command"
+require_relative "mixins/client_helper"
+require_relative "mixins/tty_helper"
+
+module Mortar
+  class ListCommand < Mortar::Command
+    include Mortar::ClientHelper
+    include Mortar::TTYHelper
+
+    option ['-q', '--quiet'], :flag, "only output shot names"
+
+    def execute
+      shots = {}
+      client.list_resources(labelSelector: LABEL).each do |resource|
+        shot_name = resource.metadata.labels&.dig(LABEL)
+        next unless shot_name # Kube API gives back some ComponentStatus resources not actually matching
+
+        if shots[shot_name]
+          shots[shot_name] += 1
+        else
+          shots[shot_name] = 1
+        end
+      end
+
+      tabs = "\t\t\t"
```

Comment:
The describe PR brings in TTY::Table so maybe use it here :)

Reply:
yeah, that was the idea :)

Reply:
done